src/main/java/TaskManager.java

```diff
@@ -0,0 +1,47 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
```

Comment:
Perhaps a more informative name would be maxTasksNo to keep in line with taskNo, or name those two constants such that they sound similar